frontend/public/locales/en/translation.json

```diff
@@ -469,6 +469,14 @@
     "contribute": "Contribute",
     "theSimpliestWayToContribute": "The simplest way to contribute to Tournesol is to compare videos. What content would you like YouTube to recommend to your friends and family?"
   },
+  "fundingSection": {
+    "supportUs": "Support Us!",
+    "tournesolExistsThanksToYourInvolvment": "Tournesol exists thanks to your involvment. By financing Tournesol you...",
+    "guaranteeTheProjectsIndependence": "<0>guarantee</0> the project's independance",
+    "participateInTheCreationOfCommons": "<0>participate</0> in the creation of commons",
```

Comment:
I don't really understand what "commons" are

Reply:
Tournesol is a digital common good of humanity, mainly because if its Free license.

I wanted to write something shorter than "digital common good", and I wondered if commons works in English.

For instance, on this [Wikipedia page](https://en.wikipedia.org/wiki/Digital_commons_(economics)) we can see:

> The digital commons are a form of [commons](https://en.wikipedia.org/wiki/Commons) [...]

So I think it works but not 100% sure.